SonarQube.Common/SarifUtilities.cs

```diff
@@ -0,0 +1,91 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="SarifUtilities.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace SonarQube.Common
+{
+    public class SarifUtilities
+    {
+        /// <summary>
+        /// Returns true if the given SARIF came from the VS 2015 RTM Roslyn, which does not provide correct output.
+        /// </summary>
+        public static bool IsSarifFromUnsupportedCompiler(string input)
+        {
+            // low risk of false positives / false negatives
+            if (input.Contains(@"""toolName"": ""Microsoft (R) Visual C# Compiler""")
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Correct the spelling of 'Sarif' in member name 'SarifUtilities.IsSarifFromUnsupportedCompiler(string)' or remove it entirely if it represents any sort of Hungarian notation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=fxcop%3AIdentifiersShouldBeSpelledCorrectly)